pkg/services/uzi/validator.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Nuts auth
+ * Copyright (C) 2020. Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package uzi
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/nuts-foundation/nuts-auth/pkg/contract"
+	"github.com/nuts-foundation/nuts-auth/pkg/services"
+)
+
+// ContractFormat is the contract format type
+const ContractFormat = contract.SigningMeans("uzi")
+
+// VerifiablePresentationType contains the string used in the VerifiablePresentation type array to indicate the Uzi means
+const VerifiablePresentationType = contract.VPType("NutsUziPresentation")
+
+// Verifier implements the Verifier interface and verifies the VerifiablePresentations of the NutsUziPresentation type.
+type Verifier struct {
+	UziValidator services.VpProofValueParser
+}
+
+// Presentation is a VerifiablePresentation without valid cryptographic proofs
```

Comment:
Strictly speaking when looking at the VP spec, a Presentation is a VP without Proof, I think?

Reply:
Copy pasted the comment from the dummy means. Made no sence in the uzi context. Fixed 👍